pkg/pillar/go.mod

```diff
@@ -4,26 +4,24 @@ go 1.12
 
```

Comment:
Why changes to go.mod @nordmark ?

Reply:
When I build it changes it for me. Don't know why. I'll revert and remove.